apps/condo/bin/lib/script-core.js

```diff
@@ -0,0 +1,52 @@
+const path = require('path')
```

Comment:
this file is redundant. You really don't reuse it. Please, don't add redundant levels of abstractions